lib/netplay/netreplay.cpp

```diff
@@ -59,14 +59,15 @@ typedef std::vector<uint8_t> SerializedNetMessagesBuffer;
 static moodycamel::BlockingReaderWriterQueue<SerializedNetMessagesBuffer> serializedBufferWriteQueue(256);
 static SerializedNetMessagesBuffer latestWriteBuffer;
 static size_t minBufferSizeToQueue = DefaultReplayBufferSize;
-static std::unique_ptr<wz::thread> saveThread;
+static WZ_THREAD *saveThread = nullptr;
```

Comment:
Is there a reason not to use `std::unique_ptr<WZ_THREAD>`?

Reply:
WZ_THREAD instances should only be created and managed by the wzThread* functions. (They wrap SDL Threads.)

Reply:
Looks like my edit stepped on your comment - we actually *want* this to leak on a non-graceful shutdown.